src/components/SingleArticle.jsx

```diff
@@ -1 +1,50 @@
-export const SingleArticle = () => {};
+import { useParams } from 'react-router';
+import { useState, useEffect } from 'react';
+import { getArticleById, getCommentsByArticleId } from '../api';
+
+export const SingleArticle = () => {
+	const params = useParams();
+	const { article_id } = params;
```

Comment:
Just a minor one, but you could save yourself a line of code here by just saying `{ article_id } = useParams()` 🙂